lib/elements/element.py

```diff
@@ -262,6 +263,119 @@ def ties(self):
     def force_lock_stitches(self):
         return self.get_boolean_param('force_lock_stitches', False)
 
+    @property
+    @param('lock_start',
+           _('Tack stitch'),
+           tooltip=_('Tack down stitch type'),
+           type='combo',
+           default='half_stitch',
+           options=LOCK_DEFAULTS['start'],
+           sort_index=52)
+    def lock_start(self):
+        return self.get_param('lock_start', "half_stitch")
+
+    @property
+    @param('lock_custom_start',
+           _('Custom svg path'),
+           tooltip=_("Enter a custom svg path. The last node will not be embroidered, but represents the first stitch of the element."),
+           type="string",
+           default="",
+           select_items=[('lock_start', 'custom')],
+           sort_index=53)
+    def lock_custom_start(self):
+        return self.get_param('lock_custom_start', '')
+
+    @property
+    @param('lock_start_scale_mm',
+           _('Scale tack stitch'),
+           tooltip=_('Set stitch length. A 1 in a custom path equals this values.'),
+           type='float',
+           unit="mm",
+           default=0.7,
+           select_items=[('lock_start', lock.id) for lock in LOCK_DEFAULTS['start'] if "absolute" in lock.path_type],
+           sort_index=54)
+    def lock_start_scale_mm(self):
+        return self.get_float_param('lock_start_scale_mm', 0.7)
+
+    @property
+    @param('lock_start_scale_percent',
+           _('Scale tack stitch'),
+           tooltip=_('Scale tack stitch by this percentage.'),
+           type='float',
+           unit="%",
+           default=100,
+           select_items=[('lock_start', lock.id) for lock in LOCK_DEFAULTS['start'] if "svg" in lock.path_type],
+           sort_index=54)
+    def lock_start_scale_percent(self):
+        return self.get_float_param('lock_start_scale_percent', 100)
+
+    @property
+    @param('lock_end',
+           _('Lock stitch'),
+           tooltip=_('Lock stitch type'),
+           type='combo',
+           default='half_stitch',
+           options=LOCK_DEFAULTS['end'],
+           sort_index=55)
+    def lock_end(self):
+        return self.get_param('lock_end', "half_stitch")
+
+    @property
+    @param('lock_custom_end',
+           _('Custom svg path'),
+           tooltip=_("Enter a custom svg path. The first node will not be embroidered, but resembles the last stitch of the element."),
```

Comment:
I think this should be "represents" instead of "resembles" to match the start one?

Reply:
Oh yes of course, I missed that one, thanks